war/src/main/js/filter-build-history.js

```diff
@@ -357,9 +358,9 @@ function checkRowCellOverflows(row) {
     }
 
     if (buildControls && !controlsRepositioned) {
-        var buildBadge = $(buildControls).getElementsBySelector('.build-badge')[0];
+        buildBadge = $(buildControls).getElementsBySelector('.build-badge')[0];
         if (buildBadge) {
-            var badgeOverflowParams = getElementOverflowParams(buildBadge);
+            badgeOverflowParams = getElementOverflowParams(buildBadge);
```

Comment:
Ah yes, the intricacies of variable scope in JavaScript… :smile: I don't understand this but I trust you.